storage_v2/v1adapter/writer.go

```diff
@@ -39,3 +42,17 @@ func (t *TraceWriter) WriteTraces(ctx context.Context, td ptrace.Traces) error {
 	}
 	return errors.Join(errs...)
 }
+
+type DependencyWriter struct {
```

Comment:
What is the intention of this type? It doesn't seem to do anything, it duplicates v1 writer API.